client/src/components/SearchBar.jsx

```diff
@@ -0,0 +1,17 @@
+import './SearchBar.css'
+
+function SearchBar() {
+  return (
+    <div className="SearchBar">
+      <input type="text" placeholder="Rechercher..." />
+      <div className="btn-search">
+        <img
+          src="../assets/images/icons/loupe-black.svg"
+          alt="Bouton de recherche"
```

Comment:
why you did not use a button with an icon instead of an img which has the comportement of a button.. not logical 🤐😶